.circleci/config.yml

```diff
@@ -110,14 +110,22 @@ jobs:
           path: results
 
 
-  image-tests-mpldev:
+  image-tests-mpl310:
     docker:
-      - image: astropy/image-tests-py36-base:1.1
+      - image: astropy/image-tests-py37-mpl310:1.5
     steps:
       - checkout
       - run:
-          name: Install apt dependencies
-          command: apt-get install -y libfreetype6-dev libpng12-dev pkg-config cm-super
```

Comment:
I suppose you'll need to keep these for the mpl build

Reply:
No these can be removed as they are in v1.2 of the docker image, but I had to add the c++ compiler below.